modules/acquia_cms_headless/acquia_cms_headless.info.yml

```diff
@@ -9,12 +9,9 @@ dependencies:
   - jsonapi_extras:jsonapi_extras
   - jsonapi_menu_items:jsonapi_menu_items
   - jsonapi_resources:jsonapi_resources
-  - next:next
   - next:next_jsonapi
   - openapi_jsonapi:openapi_jsonapi
-  - openapi_ui:openapi_ui
   - openapi_ui_redoc:openapi_ui_redoc
   - openapi_ui_swagger:openapi_ui_swagger
   - drupal:rest
```

Comment:
@apathak18 also you can remove '- drupal:rest' as restui has added this dependency.